dbschema/project.esdl

```diff
@@ -60,6 +60,24 @@ module default {
         message := "A Project's own context should be itself (no more or less)"
       )
     );
+
+    trigger createBudgetOnInsert after insert for each do (
+      insert default::Budget {
+        project := __new__.project,
+        projectContext := __new__.projectContext,
+      }
+    );
+
+    trigger updateBudgetSensitivityOnSensitivityChange after update for each do (
```

Comment:
No need. The `projectContext` given on insert calculates the `sensitivity` on read.
https://github.com/SeedCompany/cord-api-v3/blob/9f47163ab06438265a7e8dff549522ab74bcc596/dbschema/project.esdl#L115-L117